projects/ngx-formentry/src/components/file-upload/file-upload.component.ts

```diff
@@ -21,6 +23,9 @@ import { DataSource } from '../../form-entry/question-models/interfaces/data-sou
 export class RemoteFileUploadComponent implements OnInit, ControlValueAccessor {
     uploading = false;
     innerValue = null;
+    resultsIsPdf = false;
+    formEntry = true;
```

Comment:
What does the formEntry flag mean and do?

Reply:
used as an input on file-uploader to prevent the upload of multiple images, yet it can only be picked as a single obs